src/toil/leader.py

```diff
@@ -496,15 +496,15 @@ def _processLostJobs(self):
             # We only rescue jobs every N seconds, and when we have apparently
             # exhausted the current jobGraph supply
             self.reissueOverLongJobs()
-            logger.info("Reissued any over long jobs")
+            logger.debug("Reissued any over long jobs")
```

Comment:
Keep this at info.  Also, potentially add a comment about toil still running (if toil takes a few hours or days, this is the message that prints every hour).